### ISSUE TITLE
TTS0001: Add speaker profile or metadata to the training data. (MM24)

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/stt-catalog-merger#8]
### Description

We need to integrate metadata from the TTS data collection into the training data. This will enhance the creation of a more balanced test set for model evaluation by ensuring the test data is evenly distributed, leading to more accurate model assessments.

We need data from STT: 
    1. [NS](https://docs.google.com/spreadsheets/d/107pF1LcHgbJtCGCdGwffqIyM70eImBmgPOjUuHQfZ7g/edit?gid=769901849#gid=769901849)
    2. [AB](https://docs.google.com/spreadsheets/d/1yKSzConuVWo8BuMDs2mabF5iiBKUz2wF--LIabFN6QE/edit?gid=0#gid=0)
    3. [TT](https://docs.google.com/spreadsheets/d/1zX1JEUigHX5gSwfJKqeerhs_5Xxkq8wFbpVUBjrKEnc/edit?gid=0#gid=0) (optional )

### Completion Criteria

Metadata from the TTS data collection is successfully incorporated into the training data.

### Subtask
- [x] get the current csv file of training data from each department. 
- [x] parse the google sheet catalog to get df of audios and it's metadata.
- [x] select list of metadata of audio to be merged. 
- [x] wrtie a script to make the meta values uniform.
- [x] merge the metadata of audio file in catalog with the corresponding audio segment that is derived from the audio. 
- [x] create a documentation of explanation of meta data values.

**Comment by gangagyatso4364** [comment]
first merge the speaker meta data with catalog sheet.

**Comment by kaldan007** [comment]
we will get tts training data with speaker metadata
